platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/config/legacy/LegacyConfigPropertiesLoader.java

```diff
@@ -117,6 +120,7 @@ public static LegacyConfigProperties loadConfigFile(@NonNull final Path configPa
                                     addressBook.add(address);
                                 }
                             } catch (final ParseException ex) {
+                                logger.info(LogMarker.CONFIG.getMarker(), "{}:{}:{}: {}", configPath, lineNumber, ex.getErrorOffset(), ex.getMessage());
```

Comment:
I suggest logging this at an error level and using the `EXCEPTION` marker. This is an error case, and we want parse exceptions to cause our tests to fail. Logging an info message is useful for debugging but will not cause the test validator to fail.
```suggestion
                                logger.error(EXCEPTION.getMarker(), "{}:{}:{}: {}", configPath, lineNumber, ex.getErrorOffset(), ex.getMessage());
```

Reply:
Perhaps while this code is being modified, we could make the log message more informative. Perhaps something like this:

```
file {}, line {}, column {}: {}
```